src/test/java/com/mathworks/ci/RunMatlabTestsBuilderTester.java

```diff
@@ -27,12 +27,12 @@
 
 public class RunMatlabTestsBuilderTester extends RunMatlabTestsBuilder {
 
-    private boolean tapChkBx;
-    private boolean junitChkBx;
-    private boolean coberturaChkBx;
-    private boolean stmResultsChkBx;
-    private boolean modelCoverageChkBx;
-    private boolean pdfReportChkBx;
+    private TapArtifact tapChkBx;
+    private JunitArtifact junitChkBx;
+    private CoberturaArtifact coberturaChkBx;
+    private StmResultsArtifact stmResultsChkBx;
+    private ModelCovArtifact modelCoverageChkBx;
+    private PdfArtifact pdfReportChkBx;

```

Comment:
Seems like these variable names need to be updated to reflect the changes you've made to their type?